code/src/main/java/frc/robot/common/Utils.java

```diff
@@ -0,0 +1,48 @@
+package frc.robot.common;
+
+import edu.wpi.first.wpilibj.geometry.Pose2d;
+import edu.wpi.first.wpilibj.geometry.Rotation2d;
+
+public class Utils {
```

Comment:
There's a Utils class in the commands folder.  Could you move that file to common and use that one?

Reply:
done